client/packages/reports/src/DetailView/DetailView.tsx

```diff
@@ -133,7 +133,10 @@ const DetailViewInner = ({
           }
         }
       } catch (error) {
-        console.error(error);
+        setState({
+          s: 'error',
+          errorMessage: t('error.failed-to-generate-report'),
+        });
```

Comment:
We were just doing console.error, so nothing showing to user. Because we never set the state, always showed loading spinner.

![Screenshot 2025-04-16 at 4 44 27 PM](https://github.com/user-attachments/assets/5b7f19a2-b0e4-480a-9a6e-f6887a5cb2d2)